source/dub/generators/build.d

```diff
@@ -624,98 +635,232 @@ struct Watcher
 			kevent_t event;
 			EV_SET(&event, fd, EVFILT_VNODE, EV_ADD | EV_CLEAR, fflags, 0, null);
 			errnoEnforce(kevent(eventfd, &event, 1, null, 0, null) != -1);
+		} else version (Windows) {
+			paths ~= path; 
 		} else
 			static assert(0, "no file watcher event system");
 	}
 
-	void wait(Pid pid)
+	version(Windows)
 	{
-		import core.sys.posix.signal, core.sys.posix.sys.select;
-
-		// Block SIGCHLD
-		sigset_t set;
-		sigemptyset(&set);
-		sigaddset(&set, SIGCHLD);
-		sigprocmask(SIG_BLOCK, &set, null);
-
-		// Add dummy handler SIGCHLD
-		extern(C) void dummy_handler(int) {}
-		sigaction_t sa;
-		sa.sa_handler = &dummy_handler;
-		sa.sa_flags = 0;
-		sigemptyset(&sa.sa_mask);
-		sigaction(SIGCHLD, &sa, null);
-
-		if (!tryWait(pid).terminated)
+		// This struct is not used at all.
+		// TODO: try to optimize it out?
+		struct OVERLAPPED {
+			size_t* Internal; size_t* InternalHigh;
+			struct Inner { uint Offset; uint OffsetHigh; }
+			HANDLE hEvent;
+		}
```

Comment:
I think this should be deleted, and give the winapi functions a dummy buffer with the right size.